### PR TITLE
Penumbra 3.0 restructuring (stage 1 of 2)

### PR DESCRIPTION
**Pull request by eligrey**
I'm working on the initial groundwork for Worker location management, which will lead into the Penumbra 3.0 API changes and functioning worker builds.

### CODE REVIEW

**Comment by eligrey** [comment]
Good news: We still support dynamic worker loading without having to hard-code static `new Worker` call signatures.

This is now ready to merge. `npm run build` now successfully builds `bundle.js`, `decrypt.penumbra.worker.js`, and `zip.penumbra.worker.js`.

Stage 2 will be to complete my implementation of `penumbra.get()` and `penumbra.zip()` and flesh out the worker offloading for those functions.

**Comment by eligrey** [comment]
airtap is currently broken. I hope to get this fixed before tackling stage 2.

File: package.json [hunk 1]
```diff
@@ -49,20 +49,24 @@
   "homepage": "https://github.com/transcend-io/penumbra#readme",
   "dependencies": {
     "crypto-browserify": "^3.12.0",
+    "comlink": "^4.0.1",
     "file-saver": "^2.0.2",
     "mime-types": "^2.1.24",
     "streamsaver": "^2.0.3",
-    "typedarray-to-buffer": "^3.1.5"
+    "typedarray-to-buffer": "^3.1.5",
+    "worker-loader": "^2.0.0"
```
Comment (bencmbrook) [inline package.json]: I think this can be a devDev

File: tsconfig.webpack.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// {
```
Comment (bencmbrook) [inline tsconfig.webpack.json]: could remove this file

**Comment by bencmbrook** [comment]
This is so awesome!

**Pull request merged**